randomkick.py

```diff
@@ -18,7 +18,13 @@
 DELAY = 24 * 60 * 60
 
 clicked = asyncio.Event()
-chosen = None
+
+try:
+    import pickle
+    chosen = pickle.loads(open('CHOSEN.pd', 'rb').read())
```

Comment:
Again, not using `with open`. Also, this will break on layer changes, which you don't handle. I think saving the user ID to a text file would be enough. Or maybe write both ID and hash.

Reply:
You only ever use the ID of it anyway.